const sendRequest = async () => {
  try {
    const response = await fetch()

    if(!response.ok) {
      
    }
  } catch (error) {
    console.error('Something went wrong', {error})
  }
}